### PR TITLE
Qwerty VKeyboard button fix( z, Q and W and ] ) on Linux(Ubuntu Focal Fossa)

### PR DESCRIPTION
**Pull request by metavinayak**
Maintainer merge checklist
* [x] Title is descriptive/clear for inclusion in release notes.
* [x] Applied a `Component: xxx` label.
* [ ] Applied the `api-deprecation` or `api-break` label.
* [ ] Applied the `release-highlight` label to be highlighted in release notes.
* [x] Added to the milestone version it was merged into.
* [ ] **Unittests** are included in PR.
* [ ] Properly documented, including `versionadded`, `versionchanged` as needed.

### CODE REVIEW

**Review by misl6 — COMMENTED** [review]
Let's wait for a complete CI run but looks good to me.

Just to confirm my thoughts, what window provider is Kivy selecting on your side?

**Comment by misl6** [comment]
Fixes:
https://github.com/kivy/kivy/issues/5707

**Comment by metavinayak** [comment]
> Let's wait for a complete CI run but looks good to me.
> 
> Just to confirm my thoughts, what window provider is Kivy selecting on your side?

Kivy is selecting 'sdl2' on my system.

The remaining details are present below:
```
[INFO   ] [Image       ] Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] [Text        ] Provider: sdl2(['text_pango'] ignored)
[INFO   ] [Window      ] Provider: sdl2
[INFO   ] [GL          ] Using the "OpenGL" graphics system
[INFO   ] [GL          ] Backend used <sdl2>
[INFO   ] [GL          ] OpenGL version <b'4.6 (Compatibility Profile) Mesa 21.2.6'>
[INFO   ] [GL          ] OpenGL vendor <b'Intel'>
[INFO   ] [GL          ] OpenGL renderer <b'Mesa Intel(R) UHD Graphics 620 (KBL GT2)'>
[INFO   ] [GL          ] OpenGL parsed version: 4, 6
[INFO   ] [GL          ] Shading version <b'4.60'>
[INFO   ] [GL          ] Texture max size <16384>
[INFO   ] [GL          ] Texture max units <32>
[INFO   ] [Window      ] auto add sdl2 input provider
```

File: kivy/data/keyboards/qwerty.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
         ["+", "+", "+", 1],    ["\u232b", null, "backspace", 2]
     ],
     "shift_2": [
-        ["\u21B9", "\t", "tab", 1.5],  ["Q", "Q", null, 1],    ["W", "W", null, 1],
+        ["\u21B9", "\t", "tab", 1.5],  ["Q", "Q", "Q", 1],    ["W", "W", "W", 1],
```
Comment (misl6) [inline kivy/data/keyboards/qwerty.json:44]:
```suggestion
        ["\u21B9", "\t", "tab", 1.5],  ["Q", "Q", "q", 1],    ["W", "W", "w", 1],
```

**Review by misl6 — APPROVED** [review]
Committed a required change.
LGTM. Thank you!

**Pull request merged**